src/applications/mhv-secure-messaging/tests/e2e/pages/GeneralFunctionsPage.js

```diff
@@ -2,6 +2,10 @@ import mockToggles from '../fixtures/toggles-response.json';
 import { Locators } from '../utils/constants';
 
 class GeneralFunctionsPage {
+  get header() {
+    return cy.get(`h1`);
```

Comment:
I think this linter warning here is for a reason. Please see if we need to take another look

Reply:
fixed

Reply:
I am still  seeing linter warning. I think it implies to use `this` or you may transform this to an arrow function

Reply:
transformed to arrow functions